### PR TITLE
feature(tour) add tour option to homepage

### PR DESCRIPTION
**Pull request by jomadoye**
#### What does this PR do?
  Adds a tour option to the home 

#### Description of Task to be completed?
As a new user logged into the application, I should see an Introductory walkthrough of how to use Reaction Commerce.

#### How should this be manually tested?
1) Sign into the application
2) Click on the tour button
3) End the tour

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#144187573 Getting Started Tour

### CODE REVIEW

File: .meteor/versions [hunk 1]
```diff
@@ -1,10 +1,10 @@
 abernix:minifier-js@1.3.19
 abernix:standard-minifier-js@1.3.19
-accounts-base@1.2.16
+accounts-base@1.2.17
 accounts-facebook@1.1.1
 accounts-google@1.1.2
 accounts-oauth@1.1.15
-accounts-password@1.3.5
+accounts-password@1.3.6
```
Comment (tonymontaro) [inline .meteor/versions]: Please revert these version numbers back if they did not affect what you did, changing them usually break the Heroku deployment.

File: imports/plugins/included/tour/client/tour.js [hunk 2]
```diff
@@ -0,0 +1,387 @@
+import introJs from "intro.js";
+import { Reaction } from "/client/api";
+import { Meteor } from "meteor/meteor";
+import { Accounts } from "/lib/collections";
+
+const tour = introJs.introJs();
+const adminTourSteps = [
+  {
+    intro: `<h2>Welcome to Reaction Commerce</h2>
```
Comment (tonymontaro) [inline imports/plugins/included/tour/client/tour.js]: Change this to Kakashi Reaction Commerce

**Review by tonymontaro — APPROVED** [review]
Good Job!

File: .meteor/versions (hunk 1 above)
Comment (jomadoye) [reply .meteor/versions]: @andela-angene I need them for my feature :-), please fix this your wayo wayo heroku deployment.

File: imports/plugins/included/tour/client/tour.js (hunk 2 above)
Comment (jomadoye) [reply imports/plugins/included/tour/client/tour.js]: would do this now.

**Pull request merged**